File: desktop/Models/Answer.cs
namespace StadiumCompany.Models;

public class Answer
{
    public int Id { get; set; }
    public int QuestionId { get; set; }
    public string Label { get; set; } = string.Empty;
    public bool IsCorrect { get; set; }
    public decimal Weight { get; set; }

    // Navigation property
    public Question? Question { get; set; }
}

File: web/app/Models/Answer.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Answer extends Model
{
    protected $table = 'answers';

    public $timestamps = false;

    protected $fillable = [
        'question_id',
        'label',
        'is_correct',
        'weight',
    ];

    protected function casts(): array
    {
        return [
            'is_correct' => 'boolean',
            'weight' => 'decimal:2',
        ];
    }

    public function question()
    {
        return $this->belongsTo(Question::class);
    }
}
